app/controllers/registers_controller.rb

```diff
@@ -19,6 +19,9 @@ def index
 
   def in_progress
     @registers = Register.available.where(register_phase: 'Alpha').sort_by_phase_name_asc.by_name
+
+    @registers_available = Register.available.where(register_phase: 'Beta')
```

Comment:
I think these instance variables can be safely deleted now (also from `pages_controller.rb`) as we get them directly in the partial now.

Reply:
As per our discussion I've referenced the model in the controller as a variable passed to the view.